src/View/ViewableData.php

```diff
@@ -293,7 +293,7 @@ public function castingHelper($field)
         }
 
         // Fall back to default_cast
-        $default = $this->config()->get('default_cast');
+        $default = self::config()->get('default_cast');
```

Comment:
Funny, this is completely outside of the scope of this PR. @sminnee @tractorcow I'll revert all of the ViewableData ones to $this->, to prevent confusion for a later fix?